src/Microsoft.SqlTools.Hosting/Extensibility/ExtensionServiceProvider.cs

```diff
@@ -77,9 +74,7 @@ public static ExtensionServiceProvider CreateFromAssembliesInDirectory(IEnumerab
 
                 try
                 {
-                    assemblies.Add(
-                        context.LoadFromAssemblyName(
-                            AssemblyLoadContext.GetAssemblyName(path)));
+                    assemblies.Add(Assembly.LoadFrom(path));

```

Comment:
What are the changes in this file for? It's unclear how they relate to the overall PR.

Reply:
Sorry, I had to remove dependency on System.Runtime.Loader as it is not available in net 472. These changes are related to that.

Reply:
Ah, and I assume this didn't show up as a problem before because we were never using this in a net472 environment? Interesting that it didn't cause a compile error.

Reply:
Probably because we are using netstandard2.0 for compilation which has the required apis for it.